core-java/src/main/java/eu/ostrzyciel/jelly/core/RdfTerm.java

```diff
@@ -0,0 +1,461 @@
+package eu.ostrzyciel.jelly.core;
+
+import eu.ostrzyciel.jelly.core.proto.v1.Rdf;
+
+public sealed interface RdfTerm {
```

Comment:
This file was here mostly as a hack to reduce redundant encapsulation in the generated proto classes. I'm not sure which parts of it are still relevant, this depends on the codegen'd code.

Reply:
Alrite, I had a look around the rest of the code and it makes sense how you implemented it here.

This will be a performance issue, because you are allocating extra intermediate objects here. We will have to revisit this later.

Please add this to the "little book of performance angst to tackle later".